oeps/processes/oep-0055-project-maintainers.rst

```diff
@@ -0,0 +1,394 @@
+at edX to measure `repository health <https://github.com/openedx/edx-repo-health>`_.
+
+The Maintainer Scrum-or-Scrums
+------------------------------
+In order to coordinate certain maintainer responsibilities including
+voting on nominated maintainers, iterating on standards, and removing
+maintainers, minimal mechanisms for coordination are required.
+
+For now, we will assume that maintainers will self-organize to
+complete the critical activities.
+
+Coordination will typically be over asynchronous channels.
+
+From time-to-time, a task force may be required to complete certain
+project-wide responsibilities like developing repository standards,
+and tools for measuring compliance to those standards.
+
+Becoming a Maintainer
+---------------------
+After the bootstrapping phase, the Open edX project will need
+processes for adding maintainers, replacing maintainers, and ensuring
+that new components are maintained.
+
+**This section should be congruent with the Core Contributor process**
+
+A new maintainer can be added in any of the following ways:
+
+* An existing maintainer can request that a new person become a
+  component maintainer either in conjunction with the existing
+  maintainer or by replacing them.  If there is no objection from
+  existing maintainers, the role will be conferred.
+* If the component is sufficiently complex to justify multiple
+  maintainers, those maintainers form a group and should organize
+  internally and present a single point of contact (like
+  teams-as-maintainers above).
+
+We are not currently limiting staff of companies from appointing,
+nominating, or voting for other employees of their company.  This
+would be impractical as, currently, single companies are locus of
+component maintenance work. This may be addressed in future revisions
+to this proposal.
+
+Unmaintained Components
+-----------------------
+In the case of unmaintained components, nominations will be made to
+the scrum-of-scrums to fill the vacancy.  If there is no objection
+from existing maintainers, the role will be conferred.
+
+If consensus cannot be reached lazily, a vote of existing maintainers
+will be taken.  Maintainers can approve a nomination with the approval
+of at least three quarters of the members.
+
+New Components
+--------------  
+In the case of new repositories, there should be a designated
+maintainer named before the repository is created.
+
+Removing a Maintainer
+---------------------
+From time-to-time it will be necessary to remove a component
+maintainer.  This might happen because:
+
+* A maintainer leaves the project or changes employment
+* A maintainer does not have the time or interest to continue in the
+  role
+* A maintainer is unreachable
+* A maintainer is not fulfilling the responsibilities of the role
+
+Trigger events for removing a maintainer would be:
+
+* A maintainer informs the project that they can no longer continue in
```

Comment:
Should we generalize the program administrator role that was introduced in the Core Contributor Process to also be the point of contact for the Project Maintainer Process?  Or should we use parallel language and create a role of "Project Maintainer Program Administrators"?

@sarina what are your thoughts on this?

Reply:
I agree. I addressed this in https://github.com/openedx/open-edx-proposals/pull/314/files